docs/LICENSE.md

```diff
@@ -0,0 +1,342 @@
+# Licenses for QUÉtudes-info
+
+The content and code of QUÉtudes-info are available under two separate licenses. For the prose and text license, see [License for all prose content](#license-for-all-prose-content). For the source code and design license, see [License for all source code and designs](#license-for-all-source-code-and-designs).
+
+## License for all prose content
+
+All prose content of QUÉtudes-info is available under [(CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/).
+
+### The CC BY-SA 4.0 license
+
+```
```

Comment:
No "unlabeled code blocks", even if it's just text (related to markdown lint rules)

```suggestion
```text
```